pkg/helm/informers.go

```diff
@@ -0,0 +1,179 @@
+package helm
+
+import (
+	"encoding/json"
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/mitchellh/hashstructure"
+	"github.com/pkg/errors"
+	apptypes "github.com/replicatedhq/kots/pkg/app/types"
+	"github.com/replicatedhq/kots/pkg/appstate"
+	appstatetypes "github.com/replicatedhq/kots/pkg/appstate/types"
+	identitydeploy "github.com/replicatedhq/kots/pkg/identity/deploy"
+	identitytypes "github.com/replicatedhq/kots/pkg/identity/types"
+	"github.com/replicatedhq/kots/pkg/kotsutil"
+	"github.com/replicatedhq/kots/pkg/logger"
+	registrytypes "github.com/replicatedhq/kots/pkg/registry/types"
+	"github.com/replicatedhq/kots/pkg/render"
+	"github.com/replicatedhq/kots/pkg/template"
+	"github.com/replicatedhq/kots/pkg/util"
+	"k8s.io/client-go/kubernetes"
+)
+
+var monitorMap map[string]*appstate.Monitor
```

Comment:
looks like this map is accessed from different go routines but is not protected by a mutex

Reply:
yup, added mutex